### DISCUSSION TITLE
[GENERAL] Editor question

### DISCUSSION THREAD

**Discussion by Selyria** [azerothcore/azerothcore-wotlk#3914] [General]
I have a question there is a DB tool like Truice z.b what is tailored to Azerothcore.

What you can edit with

Smart_Ai
creatures
quest
Gameobject

So like Truice, that was really mega good.

Thanks for the info already once

**Comment by Helias** [comment]
There is [Keira2](https://github.com/Helias/Keira2) but it isn't compatible 100% with AzerothCore.
Me and @FrancescoBorzi are planning to make a new version of Keira that will be better and full compatible with AzerothCore.

Keira2 support **SmartAI, Creature, Quest, Gameobject, Item** and more.

**[Here a demo](http://shinworld.altervista.org/Keira2/#/)**
